end-to-end-test/shared/specUtils.js

```diff
@@ -699,6 +699,42 @@ function selectClinicalTabPlotType(type) {
     ).click();
 }
 
+function clickColumnSelectionButton(patientTable) {
+    $(`[data-test=${patientTable}]`)
+        .$('button*=Columns')
+        .click();
+}
+
+function selectColumn(namespaceColumn1) {
+    $(`[data-id="${namespaceColumn1}"]`).click();
```

Comment:
any reason to introduce a new attribute instead of just using data-test?

Reply:
this is the attribute name as generated by the column picker